app/assets/javascripts/comments.js

```diff
@@ -1,5 +1,29 @@
 $( function () {
-  $('.edit_comment #comment_content').live('change', RescueRails.saveParentForm);
+
+  if ( $('.edit_comment').length > 0) {
+
+    $(document).on('click', '.toggle-edit-comment', function(){
+      var $parent = $(event.target).parents('form.edit_comment');
```

Comment:
event is a magic var in chrome or something, get that out!  the event is passed into the function, so

```
 function(e) {
 $(e.target)
````